src/main/webapp/resources/js/components/samples/components/SampleFiles.tsx

```diff
@@ -130,8 +130,7 @@ export default function SampleFiles() {
           total: number;
         }) => {
           if (progressEvent.loaded === progressEvent.total) {
-            setSequenceFiles([]);
-            setSeqFileProgress(0);
```

Comment:
Do yo think this can be simplified also through the createApi?

Reply:
I will need to take a look again but I think I saw something a while back about the AbortController (to stop the upload) doesn't work well with it

Reply:
Sounds good.